iree/turbine/kernel/boo/conv_exports/launch.py

```diff
@@ -173,6 +234,16 @@ def get_launchable(
             parameter_providers=(),
             entry_point=f"{func_name}$async",
         )
+    elif Path(BOO_TUNING_SPEC_PATH).is_file():
```

Comment:
I would explicitly check for empty string here, so file name typos don't get silently ignored
```suggestion
    elif BOO_TUNING_SPEC_PATH != "":
```